DFC.Integration.AVFeed/DFC.Integration.AVFeed.Service.AVAPI/AVAPIService.cs

```diff
@@ -36,40 +36,38 @@ public AVAPIService(IApprenticeshipVacancyApi apprenticeshipVacancyApi, IApplica
             return JsonConvert.DeserializeObject<ApprenticeshipVacancyDetails>(responseResult);
         }
 
-
         /// <summary>
         /// Get Apprenticeship Vacancy Summaries untill we have at leat two providers or all of them for the mapping
         /// </summary>
         /// <param name="mapping"></param>
         /// <returns></returns>
-        /// 
+        ///
         public async Task<IEnumerable<ApprenticeshipVacancySummary>> GetAVsForMultipleProvidersAsync(SocMapping mapping)
         {
             if (mapping == null)
             {
                 throw new ArgumentNullException(nameof(SocMapping));
             }
 
-            List<ApprenticeshipVacancySummary> avSummary = new List<ApprenticeshipVacancySummary>();
-
+            var avSummary = new List<ApprenticeshipVacancySummary>();
             var pageNumber = 0;
             var maxPagesToTry =  int.Parse(ConfigurationManager.AppSettings.Get("FAA.MaxPagesToTryPerMapping"));
 
             logger.Trace($"Getting vacancies for mapping {JsonConvert.SerializeObject(mapping)}");
-
             //Allways break after a given number off loops
             while (maxPagesToTry > pageNumber)
             {
-                var apprenticeshipVacancySummaryResponse = await GetAVSumaryPageAsync(mapping, ++pageNumber);
+                    var apprenticeshipVacancySummaryResponse = await GetAVSumaryPageAsync(mapping, ++pageNumber);
 
-                logger.Trace($"Got {apprenticeshipVacancySummaryResponse.TotalReturned} vacancies of {apprenticeshipVacancySummaryResponse.TotalMatched} on page: {pageNumber} of {apprenticeshipVacancySummaryResponse.TotalPages}");
+                    logger.Trace(
+                        $"Got {apprenticeshipVacancySummaryResponse.TotalReturned} vacancies of {apprenticeshipVacancySummaryResponse.TotalMatched} on page: {pageNumber} of {apprenticeshipVacancySummaryResponse.TotalPages}");
 
-                avSummary.AddRange(apprenticeshipVacancySummaryResponse.Results);
+                    avSummary.AddRange(apprenticeshipVacancySummaryResponse.Results);
```

Comment:
This variable may be null as you are returning null in GetAVSumaryPageAsync method, but no null checks, clearly no unit tests either, please can you update.